frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -1410,12 +1411,16 @@ test("begins_typeahead", ({override, override_rewire}) => {
     assert_typeahead_equals("<timerandom", false);
     assert_typeahead_equals("<time", ["translated: Mention a time-zone-aware time"]);
     assert_typeahead_equals("<time:", ["translated: Mention a time-zone-aware time"]);
+    assert_typeahead_equals(":octopus: <time:", ["translated: Mention a time-zone-aware time"]);
```

Comment:
It'd also be good to have more test cases for what comes before the "<time" string.

For example, I believe you noticed that using "@" might also trigger the bug.

This tokenization function actually has other buggy behaviors, but probably better handled as a different Issue.

Reply:
Good call. I added one for that, I can add more but there's an existing bug I think we need to discuss that's getting hit with some of the tests I added. (That discussion on the other comment, since that's where you mention one of the cases.)